### PR TITLE
Feed trail image

### PR DESCRIPTION
**Pull request by Reettaphant**
## What's changed?

Displays article trail pics in capi feed, reduces the clipboard size to make way for a slightly wider feed and adjusts some padding for the same purpose as well as per @akemitakagi's instructions.

## Implementation notes
Screenshot 2019-03-11 at 15 19 16

_Include any specific areas you want to highlight for review that you feel might be worthy of discussion (i.e. any non-obvious decisions you've made)_

### General
- [x] 🤖 Relevant tests added
- [ ] ✅ CI checks / tests run locally
- [x] 🔍 Checked on CODE

### Client
- [x] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [x] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [x] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

File: client-v2/src/components/FrontsCAPIInterface/Feed.tsx [hunk 1]
```diff
@@ -28,31 +29,24 @@ const Feed = ({ articles = [], error }: FeedProps) => (
...
+            publicationDate={article.webPublicationDate}
+            sectionName={article.sectionName}
+            pillarId={article.pillarId}
+            internalPageCode={
+              article.fields && getId(article.fields.internalPageCode)
+            }
+            firstPublicationDate={article.fields.firstPublicationDate}
+            isLive={!article.fields.isLive || article.fields.isLive === 'true'}
+            scheduledPublicationDate={article.fields.scheduledPublicationDate}
+            thumbnail={getThumbnail(article, article.frontsMeta.defaults)}
```
Comment (Reettaphant) [inline client-v2/src/components/FrontsCAPIInterface/Feed.tsx]: Is there a way to restructure an object and still have access to it in the fuction? I couldn't figure out the syntax if it exists.

File: client-v2/src/shared/components/Thumbnail.tsx [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+const ThumbnailBase = styled('div')`
   background-size: cover;
   background-color: ${({ theme }) =>
     theme.shared.base.colors.backgroundColorFocused};
 `;
+
+const ThumbnailSmall = styled(ThumbnailBase)`
+  width: 83px;
+  height: 50px;
+  margin: 3px 5px 3px 5px;
```
Comment (Reettaphant) [inline client-v2/src/shared/components/Thumbnail.tsx]: We should probably make an image component which can have different widths but preserves correct aspect rations and handles loading states consistently but that is perhaps for an another pr.

Comment (jonathonherbert) [reply client-v2/src/shared/components/Thumbnail.tsx]: 👍 great idea.

File: client-v2/src/components/FrontsCAPIInterface/Feed.tsx (hunk 1 above)
Comment (jonathonherbert) [reply client-v2/src/components/FrontsCAPIInterface/Feed.tsx]: not in the function params, I think -- you'd have to pass it in whole and destructure within the function body.

**Comment by jonathonherbert** [comment]
@Reettaphant looks good, it'll be nice to see the trail pics in there!

**Pull request merged**